sdk/tests/iot/common/test_az_iot_common.c

```diff
@@ -145,6 +145,48 @@ static void test_az_iot_message_properties_next_NULL_out_value_fail(void** state
   ASSERT_PRECONDITION_CHECKED(az_iot_message_properties_next(&props, &name, NULL));
 }
 
+static void test_az_iot_message_properties_next_written_less_than_size_succeed(void** state)
+{
+  (void)state;
+  #define STRSIZE sizeof(TEST_KEY_VALUE_THREE)/sizeof(TEST_KEY_VALUE_THREE[0])
```

Comment:
nit: any reason to not have this just be a var

Reply:
No specific reason at all.

Reply:
just looks a little odd to have in a function